pylint_odoo/test_repo/broken_module/models/broken_model.py

```diff
@@ -30,8 +30,11 @@ class TestModel(models.Model):
     )
 
     def my_method1(self, variable1):
-        #  Shouldn't show error of field-argument-translate
+        "Shouldn't show error of field-argument-translate"
         self.my_method2(_('hello world'))
 
     def my_method2(self, variable2):
+        '''
```

Comment:
Yes,
We will can re-use this same method to create a new check `broken_module` has many lint issues for this matter.